server/src/schemas/trainingSchema.ts

```diff
@@ -1,5 +1,30 @@
 import { gql } from "apollo-server-express";
 
+export interface TrainingModule {
+  id: number;
+  name: string;
+}
+
+export enum QuestionType {
+  MULTIPLE_CHOICE,
+  CHECKBOXES
+}
+
+export interface Question {
+  id: number | undefined;
+  moduleId?: number;
+  text: string;
+  type: QuestionType;
+  order: number;
+}
```

Comment:
Looks to me like `Question` is missing `options: Option[];` or similar

Reply:
Our repos dont actually return related collections, graphql resolves it for us instead. It makes our repos and mappers much simpler to implement/maintain